openmetadata-ui/src/main/resources/ui/src/components/Settings/Services/AddService/Steps/SelectServiceType.tsx

```diff
@@ -146,11 +144,7 @@ const SelectServiceType = ({
                 {BETA_SERVICES.includes(
                   type as DatabaseServiceType | PipelineServiceType
                 ) ? (
-                  <Badge
-                    className="service-beta-tag"
-                    color={theme.primaryColor}
-                    count={t('label.beta')}
-                  />
+                  <Badge className="service-beta-tag" count={t('label.beta')} />
```

Comment:
@Ashish8689 , what is the reason for not using the theme color?

Reply:
we already have the styling with the class `service-beta-tag`. so not needed here!